examples/gno.land/r/gnoland/faucet/faucet_test.gno

```diff
@@ -5,18 +5,25 @@ import (
 	"std"
 	"testing"
 
-	"gno.land/p/testutils"
-	"gno.land/r/faucet"
+	"gno.land/p/demo/testutils"
+	"gno.land/r/gnoland/faucet"
 )
 
 func TestPackage(t *testing.T) {
 	var (
-		adminaddr       = std.Address("g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5")
-		faucetaddr      = std.TestDerivePkgAddr("gno.land/r/faucet")
-		controlleraddr1 = testutils.TestAddress("controller1")
-		controlleraddr2 = testutils.TestAddress("controller2")
-		controlleraddr3 = testutils.TestAddress("controller3")
-		controlleraddr4 = testutils.TestAddress("controller4")
+		adminaddr        = std.Address("g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5")
+		faucetaddr       = std.TestDerivePkgAddr("gno.land/r/faucet")
+		controlleraddr1  = testutils.TestAddress("controller1")
+		controlleraddr2  = testutils.TestAddress("controller2")
+		controlleraddr3  = testutils.TestAddress("controller3")
+		controlleraddr4  = testutils.TestAddress("controller4")
+		controlleraddr5  = testutils.TestAddress("controller5")
+		controlleraddr6  = testutils.TestAddress("controller6")
+		controlleraddr7  = testutils.TestAddress("controller7")
+		controlleraddr8  = testutils.TestAddress("controller8")
+		controlleraddr9  = testutils.TestAddress("controller9")
+		controlleraddr10 = testutils.TestAddress("controller10")
+		controlleraddr11 = testutils.TestAddress("controller11")
```

Comment:
What do you think about defining a helper method for this test code?
We can reuse it for other use cases as well.

I'm not sure on the return element type, but I'm guessing it's `crypto.Address`

```go
// generateTestAddresses generates a collection of test addresses
func generateTestAddresses(prefix string, count int) []crypto.Address {
    addrs := make([]crypto.Address, count)
    
    for i := 0; i < count; i++ {
        addrs[i] = testutils.TestAddress(fmt.Sprintf("%s%d", prefix, i))
    }

    return addrs
}

```

Reply:
This is good; we can write a utility like this.  

However, in this case, I purposely named the controller starting from 1, not 0.  Using the 0-based indexing slice is not straightforward for testing the boundary case.

The testing code should be straightforward and look dumb.  The ten controllers are more than enough for a faucet in our use case.  Dynamic value and extra abstraction only complicate the testing case.   We can quickly identify the boundary case by reading the name used to identify it instead of using dynamically generated values with indexes, which could introduce bugs or cause us to use the wrong address. 

Also,  gno does not support fmt.Sprintf yet.